.github/workflows/ci.yml

```diff
@@ -1,5 +1,12 @@
 name: tests
-on: push
+
```

Comment:
This prevents the test workflow running when a release is made but enables it for PRs and everything on the main branch.